sections/futures/UserInfo/UserInfo.tsx

```diff
@@ -103,6 +107,17 @@ const UserInfo: React.FC<UserInfoProps> = ({ marketAsset, position, openOrders,
 		setOpenProfitCalcModal(!openProfitCalcModal);
 	}, [openProfitCalcModal]);
 
+	const refetchTrades = useCallback(() => {
+		futuresTradesQuery.refetch();
+		marginTransfersQuery.refetch();
+	}, [futuresTradesQuery, marginTransfersQuery]);
+
+	useEffect(() => {
+		refetchTrades();
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@koredefashokun Is this a bad pattern? I noticed you used this for the open order refetching, wanted to make sure we aren't using hooks completely wrong.